src/constants/stellar-types.js

```diff
@@ -0,0 +1,5 @@
+export const STELLAR_TYPES = {
```

Comment:
You duplicate const from the component, place use one

Reply:
done